<launch>
<param name="robot_description" command="$(find xacro)/xacro --inorder '$(find learning_urdf_simulations)/urdf/Rviz_M.A.R.S_prototype2.xacro'"/>

<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher"/>
<node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>

</launch>